content/post/kubedb-v2022.05.24/index.md

```diff
@@ -0,0 +1,253 @@
+    minPoolSize: 0
+    maxClientConnections: 20
+    reservePoolSize: 5
+    maxDBConnections: 0
+    maxUserConnections: 0
+    statsPeriodSeconds: 60
+    authType: md5
+    adminUsers:
+    - myuser
+    authUser: myuser
+  userListSecretRef:
+    name: db-user-pass
+  terminationPolicy: WipeOut
+ ```
+
+## Redis
+
+We have redesigned the KubeDB Redis operator. We have distributed responsibility across different components so that there is no single point of failure. The primary task of the KubeDB Redis operator is to provision the cluster according to the provided configuration. After the KubeDB Redis operator successfully provisions the cluster, it no longer depends on the operator. The pods of the cluster can rejoin the cluster by themselves in case of node failure or network issues. We have fixed some minor bugs in KubeDB Redis Operator.
+
+## MariaDB
+
+For KubeDB MariaDB, better health checking has been added. KubeDB MariaDB CR now supports more accurate status conditions.
+
+## Schema Manager
+
+We have added `Multi-Tenancy` support for `MariaDB` in this latest release. With schema-manager now you can create a database, initialize a database using a script, restore a snapshot  for MariaDB as well. So, KubeDB now supports managing schemas with declarative YAML for MariaDB.
+
+Sample MariaDBDatabase YAML:
+
+```yaml
+apiVersion: schema.kubedb.com/v1alpha1
+kind: MariaDBDatabase
+metadata:
+ name: sample-mariadb-schema
+ namespace: demo
+spec:
+ database:
+   serverRef:
+     name: mariadb-server
+     namespace: dev
+   config:
+     name: demo
+     characterSet: utf8
+     encryption: disable
+     readOnly: 0
+ vaultRef:
+   name: vault
+   namespace: dev
+ accessPolicy:
+   subjects:
+     - kind: ServiceAccount
+       name: "tester"
+       namespace: "demo"
+   defaultTTL: "10m"
+ deletionPolicy: "Delete"
+```
+
+The given YAML will create a MariaDB database schema in the cluster which will eventually end up creating a database demo inside  mariadb-server which is a KubeDB MariaDB server. The demo database will be created with characterset `utf8` and `encryption disabled`  as it is mentioned in the `spec.database.configuration` section. The operator will create a user and its credentials using the `KubeVault` and bind the permission under the subject which is mentioned in the `spec.accessPolicy.subjects` section. The credential and permission will be valid for 10 minutes. The time duration is  mentioned in the `spec.accessPolicy.defaultTTL` section. After that, the schema will be expired.
+
+## Elasticsearch
+
+In this release, We are happy to introduce the support for `Elasticsearch V8`. Now, you can provision and manage Elasticsearch V8 with KubeDB. New supported ElasticsearchVersion CROs are `xpack-8.2.0`, `kubedb-xpack-8.2.0`, and `opensearch-1.3.2`. The `kubedb-xpack-8.2.0` packed with  `elasticsearch:8.2.0` with pre-installed plugins; `repository-s3`, `repository-azure`, `repository-hdfs`, and `repository-gcs`.
```

Comment:
The `kubedb-xpack-8.2.0` is packed with  `elasticsearch:8.2.0` with pre-installed plugins: `repository-s3`, `repository-azure`, `repository-hdfs`, and `repository-gcs`.